sdk/ai/azure-ai-projects/samples/inference/sample_get_chat_completions_client_with_tracing.py

```diff
@@ -30,6 +30,14 @@
     conn_str=os.environ["PROJECT_CONNECTION_STRING"],
 ) as project_client:
 
+
+    if not project_client.diagnostics.enable():
+        print("Application Insights was not enabled for this Azure AI Studio project.")
```

Comment:
Should we add instructions how to enable?

Reply:
Thanks, I updated the message to say that you need to go to the 'Tracing' tab under 'Tool' in AI Studio Project page.